### PR TITLE
added jquery functionality to the admin panel, fixed broken header ja…

### PR DESCRIPTION
**Pull request by SStrausman**
…vascript, added enable JS tag to the layout.
@thewildpendulum @KaraAJC

### CODE REVIEW

File: app/assets/javascripts/administration.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+$(document).ready(function(){
+    $("ul#tabs li").click(function(e){
+        e.preventDefault();
```
Comment (thewildpendulum) [inline app/assets/javascripts/administration.js:3]: I don't think you need this

File: app/assets/javascripts/administration.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+$(document).ready(function(){
+    $("ul#tabs li").click(function(e){
```
Comment (thewildpendulum) [inline app/assets/javascripts/administration.js:2]: You can change this selector to `#tabs li`

File: app/assets/javascripts/administration.js (hunk 1 above)
Comment (SStrausman) [inline app/assets/javascripts/administration.js:3]: You mean the prevent default? Yeah, it was added when I first was trying to debug why it wasn't working.

File: app/assets/javascripts/administration.js (hunk 2 above)
Comment (SStrausman) [inline app/assets/javascripts/administration.js:2]: Done.

File: config/initializers/flicker.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ENV["FLICKR_KEY"] = 'b2aef871a11feaaacea0b0095199ff81'
```
Comment (sowiecki) [inline config/initializers/flicker.rb:1]: Please 'git reset --soft HEAD~1' to undo this commit, remove the API keys, add this file to .gitignore, recommit and repush. We do not want this visible on the repo.

File: app/assets/javascripts/administration.js (hunk 1 above)
Comment (SStrausman) [inline app/assets/javascripts/administration.js:3]: Removed prevent default.

File: config/initializers/flicker.rb (hunk 3 above)
Comment (SStrausman) [inline config/initializers/flicker.rb:1]: Closing pull request. New one will have these changes.

File: app/assets/javascripts/administration.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        e.preventDefault();
+        if (!$(this).hasClass("active")) {
```
Comment (thewildpendulum) [inline app/assets/javascripts/administration.js:4]: Store a reference to `$(this)`. `var $this = $(this);`

**Pull request closed without merging**